packages/generator/src/operations/request-builder-arguments.ts

```diff
@@ -12,7 +12,7 @@ export function getRequestBuilderArguments(
   const sharedParameters = [getTransformer(operation), 'params'];
   if (operation.isBound) {
     return [
-      'this._entityApi',
+      `this._entityApi as ${operation.entityClassName}Api<T>`,
```

Comment:
[q] Why is this needed? Of course without `as` it would be nicer, if possible.

Reply:
Without explicit typecast, I got an error:
 ```
Type 'DefaultDeSerializers' is not assignable to type 'T'.
          'DefaultDeSerializers' is assignable to the constraint of type 'T', but 'T' could be instantiated with a different subtype of constraint 'DeSerializers<any, any, any, any, any, any, any, any, any, any, any, any, any, any, any, any, any, any, any>'.
```

Reply:
I think this is an issue with how the `defaultDeSerialisers` are used here. Instead of passing them around as `deSerializers || defaultDeserializers` we could add this `deSerializers: T = defaultDeSerializers as T` to the parameters instead of `deSerializers?: T`. We still have an explicit type cast, but it is on a more defined property and thus more controllable (less error prone).